DB.md

```diff
@@ -0,0 +1,87 @@
+# Some notes on interacting with postgresql database
+
+## postgresql extensions
```

Comment:
I would add caps at the beginning of your titles

Reply:
done